sphinxcontrib/inmanta/config.py

```diff
@@ -253,7 +253,12 @@ def _get_namespaces(self):
     def run(self):
         namespaces = self._get_namespaces()
         for namespace in namespaces:
-            importlib.import_module(namespace)
+            try:
+                importlib.import_module(namespace)
+            except ModuleNotFoundError:
+                # The documentation lists all possible modules with config. Only the ones that exist will be
+                # rendered. This allows a single file for both OSS and ISO releases.
+                LOGGER.warning("Unable to load module %s, no config will be loaded for this module", namespace)
```

Comment:
The first time I read this comment, I thought it was about Inmanta modules. But from the context I derived that it was a Python module. I can imagine this can be very confusing, especially for new developers.
```suggestion
                # The documentation lists all possible Python modules in the server codebase with config. Only the ones that exist will be
                # rendered. This allows a single file for both OSS and ISO releases.
                LOGGER.warning("Unable to load module %s, no config will be loaded for this module", namespace)
```